File: remote_config.js
const admin = require("firebase-admin");

admin.initializeApp();

// Get the template
admin.remoteConfig().getTemplate()
  .then(template => {
    console.log("ETag:", template.etag);

    // Get a parameter
    console.log(template.parameters.header_text.defaultValue.value);

    // Get a condition
    console.log(template.conditions[0].name);
    console.log(template.conditions[0].expression);
    console.log(template.conditions[0].tagColor);

    // Publish the template
    admin.remoteConfig().publishTemplate(template)
      .then(updatedTemplate => {
        console.log("Template has been published");
        console.log("ETag:", updatedTemplate.etag);
      });
  });

File: remote_config.py
import firebase_admin
from firebase_admin import credentials
from firebase_admin import remoteconfig

# Initialize the app
c_cred = credentials.Certificate("service-account.json")
firebase_admin.initialize_app(c_cred)

# Get the template
template = remoteconfig.get_template()

# See the ETag of the template
print('ETag:', template.etag)

# Get a parameter
print(template.parameters["header_text"].default_value.value)

# Get a condition
print(template.conditions[0].name)
print(template.conditions[0].expression)
print(template.conditions[0].tag_color)

# Publish the template
remoteconfig.publish_template(template)
